src/components/dashboard/container/SpecificConcept.jsx

```diff
@@ -15,15 +18,17 @@ export class SpecificConcept extends Component {
       owner: PropTypes.string,
     })).isRequired,
     isFetching: PropTypes.bool.isRequired,
+    clearConcepts: PropTypes.func.isRequired,
+    hasMore: PropTypes.bool.isRequired,
   };
   constructor(props) {
     super(props);
 
     this.state = {
       searchInput: '',
+      offset: 2,
     };
-    this.onSearch = this.onSearch.bind(this);
-    this.onSubmit = this.onSubmit.bind(this);
+    autoBind(this);
```

Comment:
I still think there is a better way of doing autoBind('functionOne', 'functionTwo')

Reply:
@mwaz I chose to use autoBind(this) to help bind the onSearch and onSubmit methods to this instance. I had not navigated the way suggested to do the binding but will explore it in future

Reply:
awesome..